src/pe-sp-tabs-launchpad/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "active": true,
+  "category": "Tabs",
+  "common-name": "Sp Tabs Launchpad",
+  "compatibility": [
+    "Jakarta",
+    "Kingston",
+    "London",
+    "Madrid"
+  ],
+  "contributors": [
+    "mrfabiop80"
```

Comment:
Please add your name here under contributors. This is my fault, the automation scripts use your GitHub username by default, instead of your name; I'll put this on the backlog to fix.

Reply:
Ok it makes sense!